internal/metarepos/storage.go

```diff
@@ -474,6 +474,11 @@ func (ms *MetadataStorage) registerLogStream(ls *varlogpb.LogStreamDescriptor) e
 
 	ms.metaAppliedIndex++
 
+	ms.logger.Info("register log stream",
+		zap.Any("lsid", ls.GetLogStreamID()),
+		zap.Any("reports", fmt.Sprintf("%+v", lm)),
```

Comment:
You can also use those expressions rather than `zap.Any`:

- `zap.Stringer("reports", lm)`
- `zap.String("reports", lm.String())`
- `zap.String("reports", fmt.Sprintf("%+v", lm))`

Reply:
Not only this, but similar things have been fixed.